.dockerignore

```diff
@@ -3,6 +3,5 @@
 agent
 build/_output
 zz_*
-scripts
```

Comment:
this now means we're copying all of our dev scripts into all of the containers that don't need them in the case of `ADD . .` Maybe we don't need to address it right now, but could be worth configuring different `.dockerignore`s

Reply:
Yes that is a valid concern.  Do we want to address it now? depending on the version of Docker available on evergreen there might be a nice way to address it: https://docs.docker.com/engine/release-notes/#19030

Reply:
The evg host has docker 19.03, so I'll have a go with this option

Reply:
@chatton I think this line has just been moved down, because it added a `/` at the end, so no change in behaviour should happen AFAIU